src/Flux.js

```diff
@@ -71,38 +71,17 @@ export default class Flux extends EventEmitter {
     }
   }
 
-  createActions(key, _Actions, ...constructorArgs) {
-    if (!(_Actions.prototype instanceof Actions) && _Actions !== Actions) {
-      if (typeof _Actions === 'function') {
-        const className = getClassName(_Actions);
-
-        throw new Error(
-          `You've attempted to create actions from the class ${className}, which `
-        + `does not have the base Actions class in its prototype chain. Make `
-        + `sure you're using the \`extends\` keyword: \`class ${className} `
-        + `extends Actions { ... }\``
-        );
-      } else {
-        const properties = _Actions;
-        _Actions = class extends Actions {};
-        assign(_Actions.prototype, properties);
-      }
-    }
-
-    if (this._actions.hasOwnProperty(key) && this._actions[key]) {
+  createActions(key, actionCreators) {
+    if (this._actions[key]) {
       throw new Error(
         `You've attempted to create multiple actions with key ${key}. Keys `
       + `must be unique.`
       );
+    } else {
+      const actions = createActions(::this.performAction, actionCreators);
```

Comment:
This seems wasteful. Shouldn't we bind `performAction` once in the constructor instead of each time an action is created? [micro-optimization]